GitExtUtils/GitUI/ThreadHelper.cs

```diff
@@ -49,7 +50,7 @@ internal set
             }
         }
 
-        public static JoinableTaskFactory JoinableTaskFactory => _joinableTaskFactory;
+        public static JoinableTaskFactory JoinableTaskFactory => _joinableTaskFactory!;
```

Comment:
:bulb: Consider using `?? throw new InvalidOperationException(...)` instead of `!`